subsys/pm/power.c

```diff
@@ -132,6 +133,7 @@ void pm_system_resume(void)
 	 */
 	if (!post_ops_done) {
 		post_ops_done = 1;
+		sys_trace_pm_power_state_exit(&z_power_state);
```

Comment:
here you should use the macros from `include/tracing/tracing_macros.h`, i.e. 

`SYS_PORT_TRACING_FUNC_EXIT(pm, power_state, ...)`

Reply:
Ok. Got it.

Reply:
Actually that doesn't make sense. SYS_PORT_TRACING_FUNC_EXIT is for tracing exit from a function. We are tracing power states. Should we define a set of SYS_PORT macros for it then or are they somehow generated?